setup.py

```diff
@@ -37,7 +37,7 @@
     "Flask-cors",
     "sphinx_rtd_theme",
     "Dill",
-    "halem==0.2.0",
+    "halem==0.3.0",
```

Comment:
does this break the openclsim notebooks from working with azure?

Reply:
Yes it does, I will fix this and create a new pull request in the notebook repo